### ISSUE TITLE
Is there anynone tried large margin finetune?

### ISSUE THREAD

**Issue by llearner** [clovaai/voxceleb_trainer#135]
I tried, but did not achieve good results like the reference.

Reference:
The IDLAB VoxSRC-20 Submission: Large Margin Fine-Tuning and Quality-Aware Score Calibration in DNN Based Speaker Verification
https://arxiv.org/abs/2010.11255

**Comment by lhanzl** [comment]
I try larger margin fune-tuning in speechbrain's recipe:
The initial loss was higher than the original, and it seemed that more epoch training was needed to make valError lower than the original.
What are the phenomena of your training?

**Comment by llearner** [comment]
> I try larger margin fune-tuning in speechbrain's recipe: The initial loss was higher than the original, and it seemed that more epoch training was needed to make valError lower than the original. What are the phenomena of your training?

The training loss rise first and then fall, the valid loss fall gradually, but the test EER always higher than the original EER.

**Comment by Jungjee** [comment]
Closing this issue since it has been inactive for six months
